.gitignore

```diff
@@ -138,4 +138,4 @@ config-local.yaml
 .cache.json
 
 # Static files for flask, include logo images but not profile pics
-static/profile_img/*
+profile_img/
```

Comment:
The profile images were not moved into the same tree as the logos during the recent refactor, so were not being included in the container. They've been moved there now, and this properly excludes them.